packages/trader/src/Modules/CFD/Components/cfd-real-account-display.jsx

```diff
@@ -157,8 +157,29 @@ const CFDRealAccountDisplay = ({
         }
     };
 
+    const isSyntheticCardVisible = () => {
+        return isLandingCompanyEnabled({ landing_companies, platform, type: 'gaming' }) || !is_logged_in;
+    };
```

Comment:
can we reuse this part of the code to prevent duplication?

Reply:
We can but I thought we may have different logic for having demo and real accounts in the future so I kept them separated